lib/common/huf.h

```diff
@@ -318,6 +326,14 @@ size_t HUF_compress1X_repeat(void* dst, size_t dstSize,
                        void* workSpace, size_t wkspSize,   /**< `workSpace` must be aligned on 4-bytes boundaries, `wkspSize` must be >= HUF_WORKSPACE_SIZE */
                        HUF_CElt* hufTable, HUF_repeat* repeat, int preferRepeat, int bmi2);
 
+/* Incorporate fast check(s) on the suspicion that this data is not compressible and back off
+   quickly to raw format if so. */
+size_t HUF_compress1X_repeat_fastCheck(void* dst, size_t dstSize,
```

Comment:
Do we need another function for this? Could we just add this variable to `HUF_compress1X_repeat()` and `HUF_compress4X_repeat()`, or rename them? I don't think these functions are used outside zstd, so we would only need one or the other.